modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Pagination.tsx

```diff
@@ -289,7 +289,13 @@ function Pagination({
 
 			<ClayLayout.SheetFooter>
 				<ClayButton.Group spaced>
-					<ClayButton onClick={handleSaveClick}>
+					<ClayButton
+						disabled={
+							pageSizesValidationError ||
+							defaultPageSizeValidationError
+						}
+						onClick={handleSaveClick}
+					>
```

Comment:
The current behavior in other FDSVM forms is to [do validation checks on save](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx#L389-L393), showing errors and not doing anything else if validation fails. The alternative is behavior here, where we disable the button if an error is detected on blur events. Both behaviors are fine by me, but we should be consistent. @ugeortiz What do you think, which behavior has better UX?